core/broadleaf-framework-web/src/main/java/org/broadleafcommerce/core/web/processor/GoogleAnalytics4Processor.java

```diff
@@ -80,11 +80,11 @@ public class GoogleAnalytics4Processor extends AbstractBroadleafTagReplacementPr
     /**
      * Global value, intentionally only retrieved as a file property NOT via the system properties service
      */
-    @Value("${googleAnalytics.masterWebPropertyId:null}")
-    protected String universalAnalyticsPropertyId;
+    @Value("${googleAnalytics4.masterWebPropertyId:null}")
+    protected String GoogleAnalytics4PropertyId;
```

Comment:
I think usual java naming for properties is to start from lowercase and use camelCase ?
 ```suggestion
    protected String googleAnalytics4PropertyId;
```